### PR TITLE
Makes raw HTML and browser datum popups UTF-8 aware

### PR DESCRIPTION
**Pull request by MrPerson**
## About The Pull Request

Adds meta UTF-8 charset tags all over the fucking place so they all display in UTF-8 mode.

## Why It's Good For The Game

Admin menus specifically really shouldn't break just because you named some guy "☺s the Clown"

### CODE REVIEW

File: code/modules/mob/dead/new_player/new_player.dm [hunk 1]
```diff
@@ -510,7 +510,7 @@
 		qdel(src)
 
 /mob/dead/new_player/proc/ViewManifest()
-	var/dat = "<html><body>"
+	var/dat = "<html><head><meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\"></head><body>"
```
Comment (AnturK) [inline code/modules/mob/dead/new_player/new_player.dm]: Why use " here ?

**Review by AnturK — COMMENTED** [review]
Does `<meta charset="utf-8">` not work properly ? It's more readable than equiv imo.

**Comment by MrPerson** [comment]
I don't know. Some of them were marked as HTML4, which would mean no it wouldn't, but I have no clue what the default is.

File: code/modules/mob/dead/new_player/new_player.dm (hunk 1 above)
Comment (MrPerson) [reply code/modules/mob/dead/new_player/new_player.dm]: I was originally doing that with all of them but then had a "wait a second" moment. Guess my regex missed this one.

File: code/modules/paperwork/filingcabinet.dm [hunk 2]
```diff
@@ -77,7 +77,7 @@
 		var/obj/item/P = contents[i]
 		dat += "<tr><td><a href='?src=[REF(src)];retrieve=[REF(P)]'>[P.name]</a></td></tr>"
 	dat += "</table></center>"
-	user << browse("<html><head><title>[name]</title></head><body>[dat]</body></html>", "window=filingcabinet;size=350x300")
+	user << browse("<html><head><meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\"><title>[name]</title></head><body>[dat]</body></html>", "window=filingcabinet;size=350x300")
```
Comment (AnturK) [inline code/modules/paperwork/filingcabinet.dm]: One more left here.

**Pull request merged**